Created toolbar and headline section

shinyenyi opened the pull request:
-Added html and css for toolbar and headline section
-Implemented all html code and styling for toolbar and headline section called for in the design template

Miguel-Shinyenyi committed:
set up project
-Add .gitignore file
Set up a linter for HTML and CSS
Add descriptive ReadMe file

Miguel-Shinyenyi committed:
Add html and css for toolbar and headline section
-Implemented all html code and styling for toolbar and headline section called for in the design template

ElsonOtake commented on index.html:
```diff
@@ -0,0 +1,35 @@
+  <section class="container">
+    <h2 class="greetings"> <span>Hey There. I’m Miguel</span></h2>
+    <p class="my-job-title"> <span>I’am a Software Developer</span></p>
+    <p class="my-job-description"><span>I can help you build a product , feature or website Look through some of my work
```
- [ ] Kindly avoid lines of code that are too long (100 characters or more) as required on [HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). That makes your code clean and improves readability.

ElsonOtake commented on index.html:
```diff
@@ -0,0 +1,35 @@
+<body>
+  <header class="header">
+    <nav class="nav-bar">
+      <span class="name">Miguel</span>
```
- [ ] Please create a `<a>` link for your logo. Usually, the logo is used for navigation purposes and is used to return to the main page. It's a good practice to adopt this convention for a better user experience(UX).

ElsonOtake commented on styles.css:
```diff
@@ -0,0 +1,112 @@
+  padding: 10px;
+  width: 100%;
+  height: 100%;
+  font-family: 'Roboto', serif;
```
- [x] Please install the fonts requested on the template. Without the correct setup, it's not possible to use these fonts. Check for more information on this [link](https://fonts.google.com/). The use of the correct fonts will create a better UI and makes your portfolio look great. Do the same thing for the other fonts from the template.

ElsonOtake commented on index.html:
```diff
@@ -0,0 +1,35 @@
+    <p class="my-job-description"><span>I can help you build a product , feature or website Look through some of my work
+        and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</span></p>
+    <div class="social-icon-links">
+      <a href="#"><img src="images/github.svg" alt="github icon"></a>
+      <a href="#"><img src="images/linkedin.svg" alt="linkedin icon"></a>
+      <a href="#"><img src="images/angellist.svg" alt="angellist icon"></a>
+      <a href="#"><img src="images/twitter.svg" alt="twitter icon"></a>
+      <a href="#"><img src="images/medium.svg" alt="medium icon"></a>
+    </div>
```
- [ ] Kindly use a `<ul>` tag to group the list of icons, and `<li>` per icon. That is semantically correct. The use of these tags and the `<a>` tag for the icons will let you connect your social networks, using href.

ElsonOtake reviewed (changes requested):
STATUS: CHANGES REQUIRED 🟡 

Hi @shinyenyi,

Good job so far! 🎉 🎉 
Nice work by the way. 🏆 🏆 
There are some issues that you still need to work on to improve your portfolio and to go to the next project but you are almost there! 🥲 🥲 

![woohoo-youre-almost](https://user-images.githubusercontent.com/77038610/154921145-1f7bb8f6-c843-4ba5-a534-6b840c600558.jpeg)

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Miguel-Shinyenyi committed:
Changes requested
-fix lines with more than 100 characters
-add link to logo
-install required google fonts
-use ul li to display social icons

ha-manel commented on index.html:
```diff
@@ -0,0 +1,39 @@
+  <header class="header">
+    <nav class="nav-bar">
+      <span class="name"><a href="#">Miguel</a></span>
+      <span class="menu-icon"><a href="#"><img src="images/menu.svg" alt="menu icon">
```
[Optional]
- [x] It's better if you remove the `a` tag around the hamburger menu because it shouldn't be an anchor element (`a` tag) as it will function like a button that opens the mobile menu ✍🏻 however, this way when you click the menu icon it will take you to home instead.

ha-manel commented on index.html:
```diff
@@ -0,0 +1,39 @@
+    </nav>
+  </header>
+  <section class="container">
+    <h2 class="greetings">Hey There. I’m Miguel</h2>
```
[Optional]

- [ ] It's better if you make this element an `h1` element instead of `h2` for semantics and to improve SEO. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

ha-manel commented on styles.css:
```diff
@@ -0,0 +1,119 @@
+
+.name a {
+  text-decoration: none;
+  padding: 10px;
```
[Optional]

- [ ] Please consider updating your padding in the header (both logo and hamburger button) to make it match the Figma design (in Figma it's 24px)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/185491021-81793680-59c8-4105-8476-116cac5bd672.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/185491121-4e3e3090-74cf-4053-8498-12dc91e28f2a.png)

ha-manel commented on styles.css:
```diff
@@ -0,0 +1,119 @@
+  justify-content: center;
+}
+
+.social-icon-links ul {
+  max-width: 250px;
+  padding-left: 24px;
+  display: flex;
+  justify-content: space-between;
+}
```
[Optional]

- [ ] Please consider removing the list default styling (the bullet points) to match the Figma design 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/185491296-770fe04f-be12-4c42-9f8f-e2dcbe11e8f0.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/185491337-d065b3d9-fb38-41d9-8cfe-369335ddddff.png)

ha-manel reviewed (approved):
# STATUS: APPROVED ✅
## Hi @shinyenyi 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**

<img src="https://media4.giphy.com/media/26DOoDwdNGKAg6UKI/giphy-downsized-medium.gif"/> 

## Highlights:
* Professional README file ✔️ 
* Proper use of GitHub flow ✔️  
* All tests passed ✔️ 

## Optional suggestions:
_Please check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
